addons/dialog/functions/fnc_create.sqf

```diff
@@ -67,6 +67,42 @@ scopeName "Main";
     private _controlType = "";
     private _rowSettings = [];
 
+    private _fnc_verifyListData = {
+        if (_values isEqualTo []) then {
+            {
+                _values pushBack _forEachIndex;
+            } forEach _labels;
+        };
+
+        _labels resize count _values;
+
+        _labels = _labels apply {
+            if (isNil "_x") then {
+                _x = _values select _forEachIndex;
+            };
+
+            _x params ["_label", ["_tooltip", ""], ["_picture", "", [""]], ["_textColor", [1, 1, 1, 1], [[]], 4]];
+
+            if !(_label isEqualType "") then {
```

Comment:
Couldn't you move this to `params` so it yells if anything has gone wrong with the data types? Yes, it wouldn't then continue but it's better than a silent error which then processes your data.

Reply:
I think this is here for the `_x = _values select _forEachIndex` above but I agree, it would be better to just move this up to the `params`.